roles/fetch-data-gsheets/tests/test.yml

```diff
@@ -0,0 +1,9 @@
+---
+- hosts: localhost
+  remote_user: root
+  roles:
+    - fetch-data-gsheets
```

Comment:
In order to test this on this directory you should create a `soft link` to the `roles` directory

Reply:
Added